textproc/hindent/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                hindent
+version             6.2.0
+revision            0
+license             restrictive/distributable
```

Comment:
please capitilize as is done is done in (most) other ports

Reply:
Okay, but this is how it appears in the official [source](https://trac.macports.org/wiki/PortfileRecipes) [file](https://github.com/macports/macports-infrastructure/blob/master/jobs/distributable_lib.tcl#L71).

https://github.com/macports/macports-infrastructure/blob/300f8a312d3ffe7c161967afdfda053a38739927/jobs/distributable_lib.tcl#L71